src/libraries/pretest.proj

```diff
@@ -58,4 +60,36 @@
                                          RuntimeGraphFiles="$(RuntimeIdGraphDefinitionFile)"
                                          TargetRuntimeIdentifier="$(PackageRID)" />
   </Target>
+
+  <UsingTask TaskName="CreateFrameworkListFile" AssemblyFile="$(InstallerTasksAssemblyPath)"/>
```

Comment:
`CreateFrameworkListFile` from `InstallerTasksAssemblyPath` should not be used, this file just hasn't been cleaned up since the sharedfx tooling moved to https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Build.Tasks.SharedFramework.Sdk/src/CreateFrameworkListFile.cs.

Filed https://github.com/dotnet/runtime/issues/34907 to clean up the leftover tasks.

My suggestion to @steveisok last week via IM:

> To get access to that task in particular, something needs to reference the sharedfx SDK, but it doesn't need to be a "full" project if that's too hard, can create a target in a NOP project that uses [MSBuild task] Outputs to return the task DLL location or something sneaky like that.

Reply:
@dagood It's not clear to me what the alternatives are.  Can you suggest a spot to put this / task(s) to use?

Reply:
The task already exists in compiled form in the sharedfx SDK, my suggestion is to reference the sharedfx SDK so the .NET SDK will download it and make it available to be used, like here:

https://github.com/dotnet/runtime/blob/952eecabbdd57540f4fe72c490c794e79c3d0cf3/src/installer/pkg/projects/Directory.Build.targets#L4-L6

Since this project's filename is `.proj` and the sharedfx SDK only does most of its modifications for `.depproj`/`.pkgproj`/`.sfxproj`/`.bundleproj`, I think it'll actually be safe to use the sharedfx SDK without the NOP project workaround I mentioned.